src/loaders/compute_tools/tool_common.py

```diff
@@ -73,33 +73,37 @@ def __init__(
         PROTOTYPE - Run a computational tool on a set of data.
 
         options:
-        -h, --help            show this help message and exit
+          -h, --help            show this help message and exit
 
         required named arguments:
-        --env {CI,NEXT,APPDEV,PROD,None}
-                              Environment used to run the tool.
-        --kbase_collection KBASE_COLLECTION
-                              KBase collection identifier name.
-        --load_ver LOAD_VER   KBase load version (e.g. r207.kbase.1).
+          --kbase_collection KBASE_COLLECTION
+                                KBase collection identifier name.
+          --source_ver SOURCE_VER
+                                Version of the source data. (e.g. 207, 214 for GTDB,
+                                2023.06 for GROW/PMI)
```

Comment:
Maybe mention that it needs to match the dir in `collectionssource`?

Reply:
👍

Reply:
:+1: